src/main/scala/gitbucket/core/util/RepoBase.scala

```diff
@@ -0,0 +1,3 @@
+package gitbucket.core.util
+
+case class RepoBase(baseUrl:String, sshAddress:Option[SshAddress])
```

Comment:
yep, can be done - see 24b9a9a12c4e1192ce0030104f397321780359d5. actually, i like that version better. thanks for the suggestion.